<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>790</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{  
    background:white;  
    color:black;  
    border:1px solid black; 
}  
  
QListWidget::item{  
    height: 195px;  
	width: 150px;  
	border-bottom: 2px solid black;
	border-right: 2px solid black;
}  
  
</string>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#FFFFFF;
border:1px solid black;</string>
   </property>
   <widget class="QPushButton" name="btn_config">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>160</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#A5A5A5;
color: white;   
border-radius: 5px;  
border: 2px groove gray; 
border-style: outset;
font:17px;
}
QPushButton:pressed{
background-color:rgb(121, 121, 121);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Config</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_takecamera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>160</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#A5A5A5;
color: white;   
border-radius: 5px;  
border: 2px groove gray; 
border-style: outset;
font:17px;
}
QPushButton:pressed{
background-color:rgb(121, 121, 121);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Take camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_searchfiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>160</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#A5A5A5;
color: white;   
border-radius: 5px;  
border: 2px groove gray; 
border-style: outset;
font:17px;
}
QPushButton:pressed{
background-color:rgb(121, 121, 121);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Search files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_manager">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>160</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#A5A5A5;
color: white;   
border-radius: 5px;  
border: 2px groove gray; 
border-style: outset;
font:17px;
}
QPushButton:pressed{
background-color:rgb(121, 121, 121);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Manager</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>160</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#A5A5A5;
color: white;   
border-radius: 5px;  
border: 2px groove gray; 
border-style: outset;
font:17px;
}
QPushButton:pressed{
background-color:rgb(121, 121, 121);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Open door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>160</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#A5A5A5;
color: white;   
border-radius: 5px;  
border: 2px groove gray; 
border-style: outset;
font:17px;
}
QPushButton:pressed{
background-color:rgb(121, 121, 121);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Setup</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>449</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{

background:#FFFFFF;
border:1px solid black;
}</string>
   </property>
   <widget class="QLabel" name="label_disksize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(0, 0, 0);
	font:18px;
	border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setup2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>100</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#A5A5A5;
color: white;   
border-radius: 5px;  
border: 2px groove gray; 
border-style: outset;
}
QPushButton:pressed{
background-color:rgb(121, 121, 121);
 border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Setup2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(0, 0, 0);
	font:18px;
	border:none;
}</string>
    </property>
    <property name="text">
     <string>Disk Usage:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(0, 0, 0);
	font:18px;
	border:none;
}</string>
    </property>
    <property name="text">
     <string>NetWork Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_networkstatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(0, 0, 0);
	font:18px;
	border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_hostip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(0, 0, 0);
	font:18px;
	border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
